modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -719,6 +738,65 @@ private void _addObjectDefinitionResources(
 		}
 	}
 
+	private Set<String> _getAccountEntryRestrictiveObjectRelationshipsNames(
+		ObjectRelationship[] objectRelationships,
+		com.liferay.object.model.ObjectDefinition
+			serviceBuilderObjectDefinition) {
```

Comment:
@devpedrotavares I think we can simplify and rename `serviceBuilderObjectDefinition` to `objectDefinition`. After doing this, please reorder the parameters alphabetically.